spec/PushController.spec.js

```diff
@@ -79,19 +79,6 @@ describe('PushController', () => {
     done();
   });
 
-  it('can throw on validateDeviceType when single invalid device type is set', done => {
```

Comment:
I'm deleting this test as it already exists

<img width="614" alt="Screen Shot 2021-02-20 at 2 52 24 PM" src="https://user-images.githubusercontent.com/9830365/108608225-40d0b480-738b-11eb-951d-724104c804b7.png">